src/status_im/contexts/wallet/send/events.cljs

```diff
@@ -25,9 +25,7 @@
  (fn [{:keys [db]} [suggested-routes timestamp]]
    (when (= (get-in db [:wallet :ui :send :suggested-routes-call-timestamp]) timestamp)
      (let [suggested-routes-data (cske/transform-keys csk/->kebab-case suggested-routes)
-           chosen-route          (->> suggested-routes-data
-                                      :best
-                                      first)]
+           chosen-route          (:best suggested-routes-data)]
```

Comment:
can you explain why we don't take the first here?

Reply:
If we only take the first one, we would be ignoring the other paths / routes. A route can have multiple paths, for example if you want to bridge 1 DAI to Optimism and the suggested routes are 0.3 DAI from Mainnet to Optimism and 0.7 from Arbitrum to Optimism. Seems we should take the whole array and was a mistake to take only the first one.

Reply:
Said that, I've added this commit to grab the first element in route in `:wallet/send-transaction` event just to not break current implementation for the time being. But I'll work on a refactor on #18938 so it supports multiple routes for the same transaction.